core/src/main/scala/org/apache/spark/api/java/JavaRDDLike.scala

```diff
@@ -436,6 +436,12 @@ trait JavaRDDLike[T, This <: JavaRDDLike[T, This]] extends Serializable {
   def first(): T = rdd.first()
 
   /**
+   * @return true if and only if the RDD contains no elements at all. Note that an RDD
+   *         may be empty even when it has at least 1 partition.
+   */
+  def isEmpty(): Boolean = rdd.isEmpty()
```

Comment:
Ah, right of course. I forgot about this issue. OK, will update MiMa and perhaps update the javadoc in `JavaRDDLike` to this effect?